src/azure/storage/mod.rs

```diff
@@ -1,10 +1,19 @@
 //! Azure Storage Singer
 
 mod signer;
+
 pub use signer::Signer as AzureStorageSigner;
+
 mod config;
+
 pub use config::Config as AzureStorageConfig;
+
 mod credential;
+
 pub use credential::Credential as AzureStorageCredential;
+
 mod loader;
+
 pub use loader::Loader as AzureStorageLoader;
+
+mod sas;
```

Comment:
As an inner package, we are unable to customize SAS parameters for controlling the SAS token. It may be a good idea to export them as public in the future.

Reply:
Maybe they can be exposed as Signer's config, can be decided later.